calibration/PHSEND/CGINS-PHSEND-P0069__20190523.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0069,CC_ea434,18103.0,source_file PHSEN-D_SAMI-pH_SN_P0069_Validation_Files_2019-05-23.zip > SAMI_P0069_220519_tris_Eavg_ValCert.pdf
+P0069,CC_ea578,97.8,
+P0069,CC_eb434,2296.0,
+P0069,CC_eb578,40427.0,
+P0069,CC_ind_off,0,
+P0069,CC_ind_slp,1,
+P0069,CC_psal,35,constant
```

Comment:
Looks good. There's a directly level between PHSEN-D_SAMI-pH_SN_P0069_Validation_Files_2019-05-23.zip and SAMI_P0069_220519_tris_Eavg_ValCert.pdf. Do we need to note that in source?

Reply:
Looks good.  I don't think we need to note the extra folder level.